### PR TITLE
[ISSUE #1416]  Delete ConsumerGroupName && Clean offsetTable

### PR DESCRIPTION
**Pull request by Liberxue**
## What is the purpose of the change

Delete ConsumerGroupName and Clean offsetTable  Contain User Create Group and System auto retry Group
Fixed ISSUE #1416

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java [hunk 1]
```diff
@@ -682,11 +682,24 @@ private RemotingCommand deleteSubscriptionGroup(ChannelHandlerContext ctx,
...
         this.brokerController.getSubscriptionGroupManager().deleteSubscriptionGroupConfig(requestHeader.getGroupName());
-
+        // only cleanConsumerOffsetList maker 
+        this.brokerController.getConsumerOffsetManager().cleanConsumerOffsetList(requestHeader.getGroupName());
         response.setCode(ResponseCode.SUCCESS);
         response.setRemark(null);
         return response;
     }
+    
+ //  private RemotingCommand cleanConsumerOffsetList(ChannelHandlerContext ctx,
```
Comment (duhenglucky) [inline broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java]: @Liberxue could you remove these useless codes?

**Comment by duhenglucky** [comment]
@Liberxue It seems that your code style check failed during CI process, so It would be better if you can fix this. and could you add some UI test to verify your code, moreover, it would be nice If you can consider forward compatibility issues, such as providing a configuration item to decide whether to completely remove.

**Comment by Liberxue** [comment]
> @Liberxue It seems that your code style check failed during CI process, so It would be better if you can fix this. and could you add some UI test to verify your code, moreover, it would be nice If you can consider forward compatibility issues, such as providing a configuration item to decide whether to completely remove.

Recently a little busy, I get this Fix

File: broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java (hunk 1 above)
Comment (Liberxue) [reply broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java]: ok has been cleaned~

**Comment by duhenglucky** [comment]
ping @Liberxue

**Comment by ShannonDing** [comment]
close due to conflict.

**Pull request closed without merging**